nlp-engine/city_extract.py

```diff
@@ -0,0 +1,77 @@
+import string
+import nltk
+from fuzzywuzzy import fuzz
+from googletrans import Translator
+from nltk.util import ngrams
+import requests
+import json
+
+translator= Translator()
+
+punct= string.punctuation
+
+r= requests.get('https://raw.githubusercontent.com/egovernments/egov-mdms-data/DEV/data/pb/Chatbot/CityNames.json')

```

Comment:
hardcoding

Reply:
This file is renamed as CityExtract.py in src. Please check that.